internal/command/launch/cmd.go

```diff
@@ -172,10 +189,17 @@ func run(ctx context.Context) (err error) {
 			}
 		}
 
+		status.TraceID = span.SpanContext().TraceID().String()
 		status.Duration = time.Since(startTime)
 		metrics.LaunchStatus(ctx, "launch", status)
 	}()
 
+	user, err := client.GetCurrentUser(ctx)
+	if err != nil {
+		return fmt.Errorf("failed retrieving current user: %w", err)
+	}
+	span.SetAttributes(attribute.String("user.id", user.ID))
```

Comment:
the collector [already](https://github.com/superfly/fly-otel-collector/blob/4ec6179058683c20934d444d0b5b16d930a88531/processors/span/processor.go#L125) sets this
```suggestion
```